actors.md

```diff
@@ -4,10 +4,27 @@ Any implementations of the Filecoin actors must be exactly byte for byte compati
 
 This spec describes a set of actors that operate within the [Filecoin State Machine](state-machine.md). All types are defined in [the basic type encoding spec](data-structures.md#basic-type-encodings).
 
-- [Init Actor](#init-actor)
-- [Storage Market Actor](#storage-market-actor)
-- [Storage Miner Actor](#storage-miner-actor)
-- [Payment Channel Broker Actor](#payment-channel-broker-actor)
+## Actor State
+
+```sh
+type ActorState union {
```

Comment:
There isnt a representation specified for these, and i'm not sure if we should actually put all the actor state types in a union. This will get tricky as we add more actors...

Reply:
So I think I tried to talk @dignifiedquire into this the other day.

It's true it might get tricky, but it also might be a perfectly reasonable exercise of schema evolution/migration stuff.  Add more actors?  Add union members.  Parsing data that has an "actor" value the current program doesn't recognize?  Well, halting is probably what we want to do anyway, neh?

Reply:
sounds reasonable to me